quirks/password-rules.json

```diff
@@ -102,6 +102,9 @@
     "girlscouts.org": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789$#!];"
     },
+    "gog.com": {
+        "password-rules": "minlength: 2;"
```

Comment:
I don’t think this is worth tracking in the project. A website requiring at least two characters in a password isn’t notable. Can you remove this?